provider/http.go

```diff
@@ -1389,6 +1390,17 @@ func (h *HTTPClient) GetTickersV2(ctx context.Context, market string) (*pb.GetTi
 	return tickers, nil
 }
 
+// GetKline returns the requested market's candles over a certain time period and resolution
+func (h *HTTPClient) GetKline(ctx context.Context, market string, from time.Time, to time.Time, resolution time.Duration) (*pb.GetKlineResponse, error) {
+	url := fmt.Sprintf("%s/api/v1/market/kline/%s?from=%s&to=%s&resolution=%s", h.baseURL, market, from.UTC().Format(time.RFC3339), to.UTC().Format(time.RFC3339), resolution.String())
```

Comment:
this functionality is V2 only please make it V2 same think for trader-api - it needs to be implemented as V2 as we re going to remove support for v1 soon